.gitignore

```diff
@@ -45,4 +45,5 @@ docs/_build/
 gtm.html
 gtm_ns.html
 
-.vscode/
+.vscode/**/*
```

Comment:
Is ignoring everything under `.vscode` with an exception of `settings.json.default` needed? Are there any other files inside inside the `.vscode` directory?

Reply:
Let me know whether you think this is needed after the tests on your local.

Reply:
follows recommended pattern.